tests/client/amo/components/TestAddonDetail.js

```diff
@@ -313,6 +317,34 @@ describe('AddonDetail', () => {
       rootNode.querySelector('.InstallButton-switch input').disabled);
   });
 
+  it('disables install switch and uses button for OpenSearch plugins', () => {
+    const AddSearchProviderSpy = sinon.stub(window.external,
```

Comment:
It probably relies on a getter but if you pass in the `window` object like `_window` (or something) then `_window.external` should trigger the getter.